### DISCUSSION TITLE
断供的能不能这样生活？

### DISCUSSION THREAD

**Discussion by jingzz1** [WeNeedHome/SummaryOfLoanSuspension#288] [Ideas]
失信前先离婚，孩子跟老婆，自己背失信，名下房车先卖了，然后能从银行贷多少款就贷，贷完就不还了，拿着这些钱去三线乡村开一家杂货店小吃店之类的，足够过一生了。

**Comment by EmberOfAsh** [comment]
老婆跑了怎么办？

**Reply by jingzz1** [reply]
> 老婆跑了怎么办？

跑就跑了吧，都这样了，还想让老婆跟着吃苦？

**Comment by llllyyyhhhh** [comment]
这是上个世纪日本人的办法了，天底下没有新鲜事。

**Reply by DDFighting** [reply]
这有种小时候以为自己发明了个东西，结果早就有这东西了的感觉

**Comment by JiahuaWang97** [comment]
房贷算夫妻共同债务，离婚也没用
